platform/qt/include/qquickmapboxgl.hpp

```diff
@@ -83,6 +87,7 @@ class Q_DECL_EXPORT QQuickMapboxGL : public QQuickFramebufferObject
         StyleNeedsSync   = 0x04,
         PanNeedsSync     = 0x08,
         BearingNeedsSync = 0x16,
+        PitchNeedsSync   = 0x32,
```

Comment:
Can we use `1 << {2,3,4...}` notation instead? I think it is easier to maintain than using raw values.